app/modules/notifications/models.py

```diff
@@ -155,26 +151,26 @@ def channels_to_send(self, digest=False):
         return channels
 
     def send_if_required(self):
-        from app.modules.emails.utils import EmailUtils
+        from app.extensions.email import Email
 
         channels = self.channels_to_send(False)
 
+        self._channels_sent = {}  # store what was sent out, if anything
         if channels[NotificationChannel.email]:
             config = NOTIFICATION_CONFIG[self.message_type]
             email_message_values = {
                 'context_name': 'context not set',
+                'sender_name': self.recipient.full_name,
+                'sender_link': '(sender link)',
             }
             email_message_values.update(self.message_values)
-            subject_template = f"email/en/{config['email_subject_template']}"
-            content_template = f"email/en/{config['email_content_template']}"
-            subject = render_template(subject_template, **email_message_values)
-            email_content = render_template(content_template, **email_message_values)
-            EmailUtils.send_email(
-                self.get_sender_email(),
-                self.recipient.email,
-                subject,
-                email_content,
+            email = Email()
+            email.recipients = [self.recipient]
+            email.template(
```

Comment:
I'm confused about this. What is the template member in the email class? It doesn't seem to be used. There is a template_name and a template_kwargs... 
I think this would be alos clearer if passed as parameters to the Email class.